syne_tune/optimizer/schedulers/hyperband_pasha.py

```diff
@@ -10,18 +10,24 @@
 # on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
-from typing import List, Dict, Any
+import itertools
+from typing import Any, Dict, List
+
 import numpy as np
-from syne_tune.optimizer.schedulers.hyperband_promotion import PromotionRungSystem
+from syne_tune.optimizer.schedulers.hyperband_promotion import \
```

Comment:
You need to run black formatting on your code. Usually, this is automatically run with every commit you do in the branch. It does not allow backslash

Reply:
I've reformatted it using black now